VoxelSniperCore/src/main/java/com/github/kevindagame/brush/SignOverwriteBrush.java

```diff
@@ -267,6 +269,31 @@ private int parseSignLineFromParam(final String[] params, final int lineNumber,
         return i;
     }
 
+    @NotNull
```

Comment:
Just wondering, Does the signOverwriteBrush work for the new hanging sign?

Reply:
Will need to try that, good point, it at least doesn't do back sides right now

Reply:
it works on hanging sings as well (tested on both Forge and Paper)